chill-scala/src/main/scala/com/twitter/chill/KryoBase.scala

```diff
@@ -102,8 +102,8 @@ class KryoBase extends Kryo {
 object Instantiators {
   // Go through the list and use the first that works
   def newOrElse(cls: Class[_],
```

Comment:
let's add a type here if we can. Can't these all be the same type?
```scala
def newOrElse[K](cls: Class[_ <: K],
  it: TraversableOnce[Class[_ <: K] => Try[ObjectInstatiator[K]],
  else: => ObjectInstatiator[K]): ObjectInstatiator[K] = {
```
should be doable, right? It would be better to have some types here.